runtime/doc/options.txt

```diff
@@ -2821,9 +2821,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	written even on filesystems which do metadata-only journaling.  This
 	will force the harddrive to spin up on Linux systems running in laptop
 	mode, so it may be undesirable in some situations.  Be warned that
-	turning this off increases the chances of data loss after a crash.  On
-	systems without an fsync() implementation, this variable is always
-	off.
+	turning this off increases the chances of data loss after a crash.
 	Also see 'swapsync' for controlling fsync() on swap files.
```

Comment:
:+1: to removing `swapsync`. Seems completely unnecessary.

Do that in this PR too then?